explorer/src/lib/dusk/components/card/Card.svelte

```diff
@@ -31,19 +20,11 @@
   class={classes}
   class:dusk-card--on-surface={onSurface}
 >
-  <header
-    class="dusk-card__header"
-    class:dusk-card__header--toggle-off={hasToggle && !isToggled}
-  >
-    {#if iconPath}
-      <Icon className="dusk-card__icon" path={iconPath} />
-    {/if}
-    <h3 class="h4">{heading}</h3>
-    {#if hasToggle}
-      <Switch onSurface bind:value={isToggled} />
-    {/if}
-  </header>
-  {#if !hasToggle || isToggled}
-    <slot />
+  {#if $$slots.header}
```

Comment:
I'm fine with removing the `header` tag, but I think it was @kieranhall who asked me to add it when I implemented the Card initially, so wanted to check with him.